Source/loadsave.cpp

```diff
@@ -727,7 +727,8 @@ void LoadMissile(LoadHelper *file)
 	missile._miPreFlag = file->NextBool32();
 	missile._miUniqTrans = file->NextLE<uint32_t>();
 	missile._mirange = file->NextLE<int32_t>();
-	missile._misource = file->NextLE<int32_t>();
+	// TODO: How to handle this?
+	//	missile._misource = file->NextLE<int32_t>();

```

Comment:
Decode the old format, fetch the monster or player based on id, get a pointer to them and set that on the missile.